handlers/session_sync_handler.go

```diff
@@ -39,30 +29,20 @@ func (h *Handler) SessionSyncHandler(w http.ResponseWriter, req *http.Request, _
 	}
 	logrus.Debugf("final list of active adapters: %+v", meshAdapters)
 	prefObj.MeshAdapters = meshAdapters
-	err = provider.RecordPreferences(req, user.UserID, prefObj)
+	err := provider.RecordPreferences(req, user.UserID, prefObj)
 	if err != nil { // ignoring errors in this context
 		logrus.Errorf("unable to save session: %v", err)
-		// http.Error(w, "unable to save session", http.StatusInternalServerError)
-		// return
```

Comment:
We have a bug in this area now. @nveenjain and @shivaylamba looking at this.

Reply:
Any open issue for this reference ?

Reply:
The behavior of the issue might have morphed slightly over time, but this is the relevant issue - https://github.com/layer5io/meshery/issues/273

Reply:
I don't think this is related to the issue. There was old error reporting which was commented and replaced, I have removed that part.